x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Analyzer.java

```diff
@@ -239,6 +238,9 @@ private static boolean containsAggregate(Expression exp) {
         return containsAggregate(singletonList(exp));
     }
 
+    private Metrics metrics() {
```

Comment:
Why not eliminate it? If it has limited usage one can just call `verifier.metrics()`